### ISSUE TITLE
when paste something, got a new empty line

### ISSUE THREAD

**Issue by onpaik** [Microsoft/vscode#49168]
Issue Type: <b>Bug</b>

```js
const a = 'a';
const c = 'c';
```
when i paste a word like 'b' to replace 'a', will got a new line ;
```js
const b = 'a';
// new line here,under uper line
const c = 'c';
```

VS Code version: Code 1.23.0 (7c7da59c2333a1306c41e6e7b68d7f0caa7b3d45, 2018-05-03T15:23:14.634Z)
OS version: Darwin x64 17.5.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-7360U CPU @ 2.30GHz (4 x 2300)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: enabled<br>rasterization: enabled<br>video_decode: enabled<br>video_encode: enabled<br>vpx_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|3, 3, 4|
|Memory (System)|8.00GB (0.03GB free)|
|Process Argv|/Applications/Visual Studio Code.app/Contents/MacOS/Electron -psn_0_4535379|
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (20)</summary>

Extension|Author (truncated)|Version
---|---|---
beautify|Hoo|1.3.0
moonscript|Lyn|1.1.9
vscode-language-pack-zh-hans|MS-|1.23.1
material-icon-theme|PKi|3.3.0
vscode-color|ans|0.4.5
snapshot-tools|asv|0.0.7
simple-react-snippets|bur|1.1.1
npm-intellisense|chr|1.3.0
vscode-eslint|dba|1.4.8
es7-react-js-snippets|dsz|1.5.2
tslint|eg2|1.0.28
vscode-npm-script|eg2|0.3.4
npm-dependency-links|her|1.0.0
vscode-npm-dependency|how|1.2.0
githd|hui|1.1.3
sublime-keybindings|ms-|4.0.0
debugger-for-chrome|msj|4.3.0
vscode-jsp|pth|0.0.2
rpa-eslint-snippets|rav|0.0.2
gitblame|wad|2.4.2

(1 theme extensions excluded)

</details>

**Comment by TobiasBreuer** [comment]
Hi, had the same issue regarding empty lines being inserted when pasting some code. It depended on different file types (i.e. in *.less files there was no issue - in *.ts files or *.json files the issue was present) which shows that it has something to do with code agnostic formatting rules.

However for me, this has been fixed by installing the update to v. 1.24.0.

**Comment by onpaik** [comment]
yeah, update to v. 1.24.0 can fix this, thks @TobiasBreuer

**Comment by rebornix** [comment]
Close this issue for now as it's fixed since 1.24 mentioned above. We can reopen it if anyone runs into it again. Thanks all for your feedback.
